extensions/moneylover/package.json

```
@@ -24,29 +27,31 @@
       "title": "Email",
       "description": "Email of your Money Lover account",
       "type": "textfield",
-      "required": true
+      "required": true,
+      "placeholder": "moneylover@example.com"
     },
     {
       "name": "password",
       "title": "Password",
       "description": "Password of your Money Lover account",
       "type": "password",
-      "required": true
+      "required": true,
+      "placeholder": "hunter2"
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.47.3",
+    "@raycast/api": "^1.79.0",
+    "@raycast/utils": "^1.16.3",
     "moneylover": "^1.1.2"
```

```suggestion
    "moneylover": "^1.2.0"
```

[v1.2.0 lib changes](https://github.com/lacherogwu/MoneyLover-api/commit/90e1b19117821e49a017a4cf116347b4832f2d77)

---

Hi @lacherogwu so the reason I didn't update the lib is because the latest version has implemented its own form of token caching. I'm not sure if it will require tweaking of the extension code to be compatible which is why I actively chose not to update the lib. Once this PR is merged I can update the lib and make sure all is fine after which the extension can be updated.
Does that make sense?

---

I think they co-work together, the lib uses caching only for authentication, then you can use caching for other things like you did with wallets